PyStemmusScope/soil_io.py

```diff
@@ -205,7 +205,7 @@ def _retrieve_latlon(file):
     return lat, lon
 
 
-def prepare_soil_data(config):
+def prepare_soil_data(config, forcing_filename):
```

Comment:
Instead of passing `forcing_filename` to this function, let's use `forcing_filename= utils.get_forcing_file(config)`